src/main/java/goblintserver/GoblintServer.java

```diff
@@ -74,6 +76,11 @@ public boolean startGoblintServer() {
                 log.error("Goblint exited with an error.");
                 return false;
             }
+
+            while(!goblintSocket.exists()) {
```

Comment:
Any alternatives for this stupidity? @sim642

Reply:
Could use `WatchService` to wait for the file creation event: https://www.baeldung.com/java-nio2-watchservice.